### PR TITLE
Promote iam_access_key and the corresponding _info module

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Promote iam_access_key and the corresponding _info module

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/iam_access_key.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    pass  # caught by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+from ansible_collections.amazon.aws.plugins.module_utils.transformation import scrub_none_parameters
+
+from ansible_collections.community.aws.plugins.module_utils.modules import AnsibleCommunityAWSModule as AnsibleAWSModule
```
Comment (abikouo) [inline plugins/modules/iam_access_key.py:137]:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
```

File: plugins/modules/iam_access_key_info.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    import botocore
+except ImportError:
+    pass  # caught by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+
+from ansible_collections.community.aws.plugins.module_utils.modules import AnsibleCommunityAWSModule as AnsibleAWSModule
```
Comment (abikouo) [inline plugins/modules/iam_access_key_info.py:82]:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
```

**Review by abikouo — CHANGES_REQUESTED** [review]

**Review by abikouo — APPROVED** [review]

File: plugins/modules/iam_access_key.py [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+    pass  # caught by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+from ansible_collections.amazon.aws.plugins.module_utils.transformation import scrub_none_parameters
+
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
```
Comment (hakbailey) [inline plugins/modules/iam_access_key.py:137]: This should make isort happy
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
from ansible_collections.amazon.aws.plugins.module_utils.transformation import scrub_none_parameters
```

File: plugins/modules/iam_access_key_info.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    import botocore
+except ImportError:
+    pass  # caught by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
```
Comment (hakbailey) [inline plugins/modules/iam_access_key_info.py:82]: This should make isort happy
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.botocore import normalize_boto3_result
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
```

File: plugins/modules/iam_access_key.py (hunk 3 above)
Comment (tremble) [reply plugins/modules/iam_access_key.py:137]: simplest option is `tox -m format` then add an extra commit

**Review by hakbailey — APPROVED** [review]

**Review by tremble — APPROVED** [review]

**Pull request merged**